src/python/turicreate/data_structures/sarray.py

```diff
@@ -4433,3 +4433,6 @@ def __deepcopy__(self, memo):
         immutable, this is identical to __copy__.
         """
         return SArray(_proxy = self.__proxy__)
+
+    def abs(self):
+        return abs(self)
```

Comment:
`return self.apply(abs)` will be faster since it will be done in parallel.

Reply:
abs(self) is actually hardcoded to simply call the self.__abs__(self) method, so this is fine.

Reply:
I didn't realize we had a `__abs__` method. Perhaps it would be better to just call that directly.

Reply:
It's a weird intersection -- most mathematical stuff is done through methods, but python has a smattering of built-ins that break that pattern.  I'm fine with both existing here.

Reply:
```
        """
        Returns the absolute value of each element of the Sarray.

        Examples
        --------
        >>> abs(tc.SArray([1, -1, -2]))
        dtype: int
        Rows: 3
        [1, 1, 2]

        >>> abs(tc.SArray([-1., -2.]))
        dtype: float
        Rows: 2
        [1.0, 2.0]
        """
```
Here's an example to do it. I cannot push to your @dhivyaaxim branch, and could you find some time to put the docstring on? Thanks!